static/js/lightbox_canvas.js

```diff
@@ -133,6 +139,19 @@ var LightboxCanvas = (function () {
                 }
             });
 
+            // event listener to handle zoom in and out from using keyboard keys z/Z and +/-
+            // in the canvas
+            document.addEventListener('keydown', function (e) {
```

Comment:
Hmm, I don't know if this is the right approach. You might want to look at `static/js/hotkey.js`, where you can add these couple keyboard shortcuts.

Reply:
Initially that was my approach, but the code for the lightbox canvas isn't written in a way that that can be done. Will need to change the way lightbox canvases are created in `lightbox.js` entirely. Hence I have done it this way.

Reply:
Ah, makes sense.

Reply:
We should add some comments documenting why this is the case, so it doesn't confuse future readers.

Reply:
@timabbott added some comments describing the reason for implementation.